addon/initializers/export-application-global.ts

```diff
@@ -0,0 +1,42 @@
+import Application from '@ember/application'
+import { classify } from '@ember/string'
+
+// @ts-ignore: because TS doesn't like this import
+import config from "../config/environment"
+
+export function initialize(application: Application): void {
```

Comment:
I was thinking that since export-application-global has been aechuved and since I can't think of someone using ember-cli-embedded without using export-application-global, maybe we could combine the two in one module rather than having a dependency on an archived module. I didn't.move far with this PR because I first wanted to get approval that is worth doing this. But we can forget about this PR, I don't mind. 🤷

Reply:
Yes, #246 moves `export-application-global` to deps so it can still be found, release 3.0.0 should work. But it's honestly a quick fix and `export-application-global` docs advise to implement something similar ourselves if we still need it. So I would also say let's leave this open.

Reply:
I have updated the PR with tests and an updated documentation